deepdrivemd/config_stream.py

```diff
@@ -0,0 +1,245 @@
+# Schema of the YAML experiment file
```

Comment:
deepdrivemd/config_stream.py Looks like mostly copy pasted code. We can reuse the existing code in deepdrivemd/config.py and then subclass the ExperimentConfig base class into StreamingExperimentConfig and add the additional parameters necessary to enable streaming/async runs.

Reply:
Done